src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -196,7 +196,7 @@ public function addSkippedTest($test, \Exception $e, $time)
 
     public function startTest($test)
     {
-        if (-2 < $this->state && ($test instanceof \PHPUnit\Framework\TestCase || $test instanceof TestCase)) {
+        if (-2 < $this->state && ($test instanceof \PHPUnit\Framework\TestCase || $test instanceof TestCase) && $test->getTestResultObject()) {
```

Comment:
Don't we want to move this new condition where `$test->getTestResultObject()` is actually used? 🤔 There are other things in this `if` that works without a test result object.

Reply:
Right now it's not obvious to me what it means when the `getTestResult()` retval is null. I tend to think this code should not be reached at all, but I may be missing something (e.g. isolated tests) and then we should only skip adding failures on case by case basis. ping @nicolas-grekas